config/locales/loa1/en.yml

```diff
@@ -13,7 +13,7 @@ en:
       warning_points:
         - You won’t have a login.gov account
         - We won't keep a record of your email address, password, and phone number
-        - You won't be able to access your benefits information with the Railroad Retirement Board
+        - You won't be able to secure access your information using login.gov
```

Comment:
Right, its a typo for securely. I just removed this file, it shouldn't have made it this far in the PR. Text has been updated in\ `loa/en.yml`